supervisord/tests/compose/supervisord.yaml

```diff
@@ -0,0 +1,7 @@
+version: '3.5'
+
+services:
+  supervisord:
+    image: "datadog/docker-library:supervisord_3_3_3"
```

Comment:
nit: not sure it's worth it but maybe we could make the version parametric to ease updates or to allow different tox env to run different versions of the docker image - feel free to ignore this